### PR TITLE
Metrics Description Changes

### PR DESCRIPTION
**Pull request by germonprez**
This is a fairly substantial change that will require some explanation: 

1) I have renamed the top-level metric label to Category
2) Each of the sections (top-level category) and lower-level items now carry goals
3) The tables now have the headers of name/question. 
4) Changes 2 and 3 are in-line with the method of Goal-Question-Metric

In doing this, we now consider the 'metric' to be to actual implementation and visualization of the name/question pairs. 

5) Some of the name/question pairs are slated for removal because they contain either specific visualization or filter related items that can be taken care of actual metric deployments. These visualizations and filters will be described in the name/question detailed descriptions (currently the long list in the activity_metrics.md file). We'll have to change that template a bit to capture the various visualizations and filters that are likely for that name/question pair. 

Once we get through this, we would follow the same template for the other Categories.

### CODE REVIEW

**Comment by germonprez** [comment]
Ooops. @GeorgLink Looks like I didn't clean up some of the Risk.md. Do you want me to reissue this PR or just ignore the Risk.md part?

**Comment by GeorgLink** [comment]
@germonprez you can make a change in your branch to remove those new (conflicting) lines and it should go away. Or replace the `3_Risk.md` file with the current master branch version.

**Comment by germonprez** [comment]
Done!

File: 2_Growth-Maturity-Decline.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Category: Growth-Maturity-Decline
 
-Goal: Describe category and the overall goal of the category
+Goal: Identify if the activity of an open source community is growing or declining and the level of maturity the project has reached.
```
Comment (GeorgLink) [inline 2_Growth-Maturity-Decline.md]: Thanks @BenLloydPearson 

How about changing the order: Identify the level of maturity the open source project has reached and whether its community is growing or declining.

Comment (BenLloydPearson) [reply 2_Growth-Maturity-Decline.md]: I agree, I've changed it.

**Comment by GeorgLink** [comment]
👍

**Comment by germonprez** [comment]
@BenLloydPearson I'll do that and thanks for adding the goals.

**Comment by GeorgLink** [comment]
We have had a few calls and plenty of time for feedback. We achieved lazy consensus and I propose we merge this after tomorrows call.

**Comment by germonprez** [comment]
@BenLloydPearson all changes made. Should be good to go on the GMD category

**Pull request merged**